pandas/tests/extension/test_sparse.py

```diff
@@ -237,6 +237,19 @@ def test_isna(self, data_missing):
     def test_fillna_no_op_returns_copy(self, data, request):
         super().test_fillna_no_op_returns_copy(data)
 
+    def test_fillna_readonly(self, data_missing):
+        # copy=False keyword is not ignored by SparseArray.fillna
+        data = data_missing.copy()
+        data._readonly = True
+
+        # by default copy=True, then this works fine
+        result = data.fillna(data_missing[1])
+        assert result[0] == data_missing[1]
+
+        # copy=False is ignored -> so same result as above
```

Comment:
this comment (in a few places) confused me until (i think) i figured out it refers to the keyword in the fillna method.  could clarify for future-me

Reply:
Looking back at it, the first comment above "copy=False keyword is not ignored by SparseArray.fillna"  also seems wrong, because `SparseArray.fillna` just completely ignores `copy`. 

Attempted to clarify this.